### PR TITLE
HTML Forms Project Complete

### PR DESCRIPTION
**Pull request by mohammadumar28**
Completed all the requirements provided in the [Odin project](https://www.theodinproject.com/courses/html5-and-css3/lessons/html-forms)

Request for a code review. :yum:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>mint.com signup page</title>
+</head>
+<body>
+    <!-- Logo -->
+    <header>
+        <div class="header">
+            <img src="images/logo.png" alt="Intuit Logo">
```
Comment (lucasmazo32) [inline index.html]: - [x] Kindly add all the other items on the top of the page. When you put all the items, do not use screenshots, instead, type the words, and only add the logos.

Source | Clone
-- | --
image | image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+                    <label for="phone">Phone <span class="font-300">(recommended)</span></label>
+                    <input type="tel" name="phone" id="phone">
+
+                    <label for="password">Password</label>
+                    <input type="password" name="password" id="password">
+
+                    <label for="confirm-password">Confirm Password</label>
+                    <input type="password" name="confirm-password" id="confirm-password">
+                </fieldset>
```
Comment (lucasmazo32) [inline index.html]: - [x] Kindly remove the confirm-password field as it is not on the source page.

Source | Clone
-- | --
image | image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            </div>
+        </div>
+    </main>
+    <!-- Footer -->
+    <footer>
+        <div class="footer text-center">
+            © 2020 Intuit, Inc. 
+            All rights reserved. Intuit, QuickBooks, QB, TurboTax, ProConnect and Mint are registered trademarks of Intuit Inc.<br>
+            Terms and conditions, features, support, pricing, and service options subject to change without notice.
+        </div>
```
Comment (lucasmazo32) [inline index.html]: - [x] Kindly add the three words above the disclaimer. Reminder: They are links!

Source | Clone
-- | --
image | image

**Comment by peterrobert** [comment]
### PULL REQUEST UNDER REVIEW BY PETER ROBERT (TSE)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+                    <label for="phone">Phone <span class="font-300">(recommended)</span></label>
+                    <input type="tel" name="phone" id="phone">
+
+                    <label for="password">Password</label>
+                    <input type="password" name="password" id="password">
+                </fieldset>
+                <!-- Submit button -->
+                <div class="submit-btn">
+                    <button type="submit" >
+                        <!-- <span class="lock-icon"></span> -->
```
Comment (peterrobert) [inline index.html]: Kindly remove the commented code. It will keep your code clean:smiley:

**Comment by mohammadumar28** [comment]
Added a sign-in button as stated in the feedback and removed the inline comments.

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Pull request merged**